flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/HiveCatalogITCase.java

```diff
@@ -503,16 +522,16 @@ public void testViewSchema() throws Exception {
 
             CatalogView catalogView =
                     (CatalogView) hiveCatalog.getTable(new ObjectPath("db1", "v1"));
-            Schema viewSchema = catalogView.getUnresolvedSchema();
+            ResolvedSchema viewSchema =
```

Comment:
I don't think we need to change the test.

Reply:
Why? We need compare with ResolvedSchema

Reply:
IIUC, this test method is meant to check the `CatalogView` gotten from method `hiveCatalog.getTable` . As the gotten schema is UnResolvedSchema, which compare with ResolvedSchema? it'll extended the test scope.

Reply:
Yes, you are right. But as my comments above, compare the Schema directly is not feasible/easy. So we compare the resolved version to compare with the expected one. I think it can cover the original requirements.

Reply:
Not understand why `compare the Schema directly is not feasible/easy.
Is there any problem if we don't change this test?

Reply:
Thanks for explaination. Make sense to me.